src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -189,27 +195,61 @@ public PerformanceAnalyzerPlugin(final Settings settings, final java.nio.file.Pa
         clusterSettingsManager.addSubscriberForIntSetting(
                 PerformanceAnalyzerClusterSettings.PA_NODE_STATS_SETTING, nodeStatsSettingHandler);
 
+        // Adding RTF Collectors if flag is enabled in performance-analyzer.properties
+        if (PluginSettings.instance().isTelemetryCollectorsEnabled()) {
+            LOG.info("Scheduling Telemetry Collectors");
+            scheduleTelemetryCollectors();
+        }
+
+        // Adding RCA Collectors if flag is enabled in performance-analyzer.properties
+        if (PluginSettings.instance().isRcaCollectorsEnabled()) {
```

Comment:
What will happen to the delta metrics if 2 collectors are running parallel in both modes?

Reply:
previous state and current state are maintained inside the collectors and diff is calculated there only. This should not be a concern since we have different collectors for RTF vs non-RTF collectors.